### PR TITLE
Wording changes of LZ Review

### PR DESCRIPTION
**Pull request by fskelly**
Customer feedback to change some of the wording on the checks to create more clarity

### CODE REVIEW

File: checklists/lz_checklist.en.json [hunk 1]
```diff
@@ -75,11 +75,11 @@
     {
       "category": "Identity and Access",
       "subcategory": "Identity",
-      "text": "If the following roles are considered, use Azure custom roles: Azure platform owner, network management, security operations, subscription owner, application owner",
+      "text": "Consider using Azure custom roles for the following key roles: Azure platform owner, network management, security operations, subscription owner, application owner",
       "guid": "f5664b5e-984a-4859-a773-e7d261623a76",
       "severity": "Medium",
       "training": "https://docs.microsoft.com/learn/modules/create-custom-azure-roles-with-rbac/",
-      "link": "https://docs.microsoft.com/azure/cloud-adoption-framework/ready/landing-zone/design-area/identity-access#prerequisites-for-a-landing-zone---design-recommendations"
+      "link": "https://docs.microsoft.com/azure/cloud-adoption-framework/ready/landing-zone/design-area/identity-access"
```
Comment (JasonBeck2) [inline checklists/lz_checklist.en.json:82]: Would you be able to update your branch to the previous link? https://docs.microsoft.com/azure/cloud-adoption-framework/ready/landing-zone/design-area/identity-access#prerequisites-for-a-landing-zone---design-recommendations

File: checklists/lz_checklist.en.json [hunk 2]
```diff
@@ -491,7 +491,7 @@
       "text": "If you need transit between ExpressRoute and VPN gateways, use Azure Route Server.",
       "guid": "ce463dbb-bc8a-4c2a-aebc-92a43da1dae2",
       "severity": "Medium",
-      "link": "https://docs.microsoft.com/azure/expressroute/expressroute-howto-coexist-resource-manager#to-enable-transit-routing-between-expressroute-and-azure-vpn"
+      "link": "https://docs.microsoft.com/azure/expressroute/expressroute-howto-coexist-resource-manager"
```
Comment (JasonBeck2) [inline checklists/lz_checklist.en.json:494]: Would you be able to update your branch to the previous link? https://docs.microsoft.com/azure/expressroute/expressroute-howto-coexist-resource-manager#to-enable-transit-routing-between-expressroute-and-azure-vpn

File: checklists/lz_checklist.en.json [hunk 3]
```diff
@@ -796,7 +796,7 @@
       "guid": "6138a720-0f1c-4e16-bd30-1d6e872e52e3",
       "severity": "Medium",
       "training": "https://docs.microsoft.com/learn/paths/secure-application-delivery/",
-      "link": "https://docs.microsoft.com/azure/cloud-adoption-framework/ready/landing-zone/design-area/resource-org-management-groups#management-groups-in-the-azure-landing-zone-accelerator"
+      "link": "https://docs.microsoft.com/azure/cloud-adoption-framework/ready/landing-zone/design-area/resource-org-management-groups"
```
Comment (JasonBeck2) [inline checklists/lz_checklist.en.json:799]: Would you be able to update your branch to the previous link? https://docs.microsoft.com/azure/cloud-adoption-framework/ready/landing-zone/design-area/resource-org-management-groups#management-groups-in-the-azure-landing-zone-accelerator

**Review by JasonBeck2 — CHANGES_REQUESTED** [review]
All the text updates look good but it appears a few extra changes came through reverting a recent update to the links. I've added inline comments for each one. I would make the changes to the files but this branch lives in your repo.

**Comment by fskelly** [comment]
Will make a plan. Very odd about the link changes. Thank you for the feedback

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

File: checklists/lz_checklist.en.json (hunk 1 above)
Comment (fskelly) [reply checklists/lz_checklist.en.json]: Should be fixed now - thank you for spotting this.

File: checklists/lz_checklist.en.json (hunk 2 above)
Comment (fskelly) [reply checklists/lz_checklist.en.json]: Should be fixed now - thank you for spotting this.

File: checklists/lz_checklist.en.json (hunk 3 above)
Comment (fskelly) [reply checklists/lz_checklist.en.json]: Should be fixed now - thank you for spotting this.

**Review by JasonBeck2 — APPROVED** [review]
Looks good!

**Pull request merged**